### PR TITLE
docs: rename all instances of helpgov to askgov 

### LINKED ISSUE #13: Rename occurrences of HelpGov to AskGov
## 1. `<title>`
image

## 2. Login page
image

### PR DESCRIPTION
**Pull request by nauynix**
Fixes #13 

**BEFORE**
image

**AFTER**
image

### CODE REVIEW

File: client/src/routes.js [hunk 1]
```diff
@@ -13,27 +13,27 @@ import { CitizenTerms, AgencyTerms, Privacy } from './components/PrivacyTerms/'
...
 const SearchResultsComponent = withPageTitle({
   component: SearchResults,
-  title: 'All Questions - HelpGov',
+  title: 'All Questions - Askgov',
 })
 
 const LoginComponent = withPageTitle({
   component: Login,
-  title: 'Log In - HelpGov',
+  title: 'Log In - Askgov',
```
Comment (LoneRifle) [inline client/src/routes.js]:
```suggestion
  title: 'Log In - AskGov',
```

File: client/src/routes.js [hunk 2]
```diff
@@ -13,27 +13,27 @@ import { CitizenTerms, AgencyTerms, Privacy } from './components/PrivacyTerms/'
...
 const PostFormComponent = withPageTitle({
   component: PostForm,
-  title: 'Ask a Question - HelpGov',
+  title: 'Ask a Question - Askgov',
 })
 
 const EditFormComponent = withPageTitle({
   component: EditForm,
-  title: 'Edit Question - HelpGov',
+  title: 'Edit Question - Askgov',
```
Comment (LoneRifle) [inline client/src/routes.js]:
```suggestion
  title: 'Edit Question - AskGov',
```

File: client/src/routes.js [hunk 3]
```diff
@@ -13,27 +13,27 @@ import { CitizenTerms, AgencyTerms, Privacy } from './components/PrivacyTerms/'
...
 const LoginComponent = withPageTitle({
   component: Login,
-  title: 'Log In - HelpGov',
+  title: 'Log In - Askgov',
 })
 
 const PostFormComponent = withPageTitle({
   component: PostForm,
-  title: 'Ask a Question - HelpGov',
+  title: 'Ask a Question - Askgov',
```
Comment (LoneRifle) [inline client/src/routes.js]:
```suggestion
  title: 'Ask a Question - AskGov',
```

File: client/src/routes.js [hunk 4]
```diff
@@ -13,27 +13,27 @@ import { CitizenTerms, AgencyTerms, Privacy } from './components/PrivacyTerms/'
...
 const HomePageComponent = withPageTitle({
   component: HomePage,
-  title: 'HelpGov',
+  title: 'Askgov',
 })
 
 const SearchResultsComponent = withPageTitle({
   component: SearchResults,
-  title: 'All Questions - HelpGov',
+  title: 'All Questions - Askgov',
```
Comment (LoneRifle) [inline client/src/routes.js]:
```suggestion
  title: 'All Questions - AskGov',
```

File: client/src/routes.js [hunk 5]
```diff
@@ -13,27 +13,27 @@ import { CitizenTerms, AgencyTerms, Privacy } from './components/PrivacyTerms/'
 
 const HomePageComponent = withPageTitle({
   component: HomePage,
-  title: 'HelpGov',
+  title: 'Askgov',
```
Comment (LoneRifle) [inline client/src/routes.js]:
```suggestion
  title: 'AskGov',
```

**Comment by barnettx** [comment]
can do a project wide search for `HelpGov` to check for any other files that need to change to `AskGov`

File: client/public/index.html [hunk 6]
```diff
@@ -4,13 +4,7 @@
...
     <link rel="icon" href="%PUBLIC_URL%/LogoGlyphMd.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-
-    <title>HelpGov</title>
+    <title>AskGov</title>
```
Comment (LoneRifle) [inline client/public/index.html:9]: Couple of things:
- Try to retain some meta description to help crawlers index the page
- [theme-color](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color) might be nice to incorporate as part of the design. Study the linked MDN doc, then consult with @prasanthkumaar as to choice of colours for normal and dark mode
```suggestion
    <meta
      name="description"
      content="Official answers from the Singapore Government"
    />

    <title>AskGov</title>
```

**Review by LoneRifle — CHANGES_REQUESTED** [review]
Batch up the changes in a single commit by going to the `Files changed` tab, then figure something out for `theme-color`

File: client/public/index.html (hunk 6 above)
Comment (nauynix) [reply client/public/index.html:9]: Form my conversation with him yesterday, there might be a new theme color in the next design overhaul, but it won't be so soon.

**Comment by nauynix** [comment]
Found more changes from `helpgov` as per @barnettx suggestion, lumped all of them into this PR

File: client/public/index.html (hunk 6 above)
Comment (LoneRifle) [reply client/public/index.html:9]: Ah okay. Leave it out for now then.

File: client/public/index.html [hunk 7]
```diff
@@ -4,13 +4,14 @@
...
     <link rel="icon" href="%PUBLIC_URL%/LogoGlyphMd.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta name="theme-color" media="(prefers-color-scheme: light)" content="white">
+    <meta name="theme-color" media="(prefers-color-scheme: dark)" content="black">
+    <title>AskGov</title>
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="The Q&A Platform for Government Related Services"
```
Comment (LoneRifle) [inline client/public/index.html]: Can I suggest this instead?
```suggestion
      content="Answers from the Singapore Government"
```

**Review by LoneRifle — APPROVED** [review]

**Pull request merged**